pkg/apis/storage/types.go

```diff
@@ -59,6 +59,10 @@ type StorageClass struct {
 	// PersistentVolumes of this storage class are created with
 	// +optional
 	MountOptions []string
+
+	// AllowVolumeExpand shows whether the storage class allow volume expand
+	// +optional
+	AllowVolumeExpand *bool
```

Comment:
Missing documentation of expected behavior when the field is missing/nil - is resize allowed of forbidden?

Reply:
`AllowVolumeExpansion` or `AllowExpansion` seem more accurate.  Was there a discussion on the naming in the proposal?

Reply:
this field should be "AllowVolumeExpansion"